### DISCUSSION TITLE
uniswap v3 error

### DISCUSSION THREAD

**Discussion by 0xlightfury** [community/community#36434] [Other Feature Feedback, Questions, & Ideas] [answered]
I forked Uniswap v3, but some token prices are not coming when swapping.
but it works on uniswap.

**Comment by kryptoDev0777** [comment, marked as answer]
hi, nice to meet you .
it is uniswap v3 api problem.
all price will come on localhost. but not working just on server.
because, uniswap API will work only localhost and uniswap.org.

**Reply by 0xlightfury** [reply]
Thanks

**Comment by starman1121** [comment]
you can see only token prices from uniswap v3 LP. but you can't see prices from uniswap v2 LP on your server
